provision-team/provision_aks.sh

```diff
@@ -129,7 +129,7 @@ fi
 echo "Creating AKS Cluster..."
 (
     set -x
-    az aks create -g $resourceGroupName -n $clusterName -l $resourceGroupLocation --node-count 3 --generate-ssh-keys -k 1.15.10 --service-principal $SP_ID --client-secret $SP_PASS
+    az aks create -g $resourceGroupName -n $clusterName -l $resourceGroupLocation --node-count 3 --generate-ssh-keys -k 1.17.11 --service-principal $SP_ID --client-secret $SP_PASS
```

Comment:
@andresravinet  Please take a look at PR https://github.com/Azure-Samples/openhack-devops-proctor/pull/331/files.  There are three changes that need to be made for a version bump.  If you can add the other two, we can run a quick test and approve. Tagging @BSamodien

Reply:
Updates to the versionin  **provision-vm/Docker/Dockerfile** and the **jenkins/install_jenkins.sh** need to be consistent to the version bump in the provision_aks.sh script